cluster/charts/rook-ceph/values.yaml

```diff
@@ -59,6 +59,13 @@ csi:
   enableRbdDriver: true
   enableCephfsDriver: true
   enableGrpcMetrics: true
+# toleration: NoSchedule
```

Comment:
could you add the comments at the same level of indentation as the others below?